main/webservices/api/v2.php

```diff
@@ -291,6 +291,39 @@
             $restApi->setMessageRead($messageId);
             $restResponse->setData(['status' => true]);
             break;
+        case Rest::CREATE_SESSION_FROM_MODEL:
+            if (empty($_POST['modelSessionId'])
+                || empty($_POST['sessionName'])
+                || empty($_POST['startDate'])
+                || empty($_POST['endDate'])) {
+                throw new Exception(get_lang('NoData'));
+            }
+            $newSessionId = $restApi->createSessionFromModel(
+                intval($_POST['modelSessionId']),
```

Comment:
We tend to validate data from external sources at the last possible moment before using it, so if you pass $_POST['modelSessionId'] to createSessionFromModel(), you should check the value of this POST variable *inside* the function, not outside.